### ISSUE TITLE
Latest pandoc in hackage (1.13.2) is incompatible with ghc 7.10

### ISSUE THREAD

**Issue by thomasdziedzic** [jgm/pandoc#2071]
It seems that HEAD is buildable with ghc 7.10 because it no longer has the constraint: ```haddock-library == 1.1.*```

I guess this issue would be a placeholder for people wondering, until a new version gets released.

```
$ cabal install pandoc
Resolving dependencies...
cabal: Could not resolve dependencies:
trying: pandoc-1.13.2 (user goal)
trying: base-4.8.0.0/installed-901... (dependency of pandoc-1.13.2)
next goal: haddock-library (dependency of pandoc-1.13.2)
rejecting: haddock-library-1.2.0 (conflict: pandoc => haddock-library>=1.1 &&
<1.2)
rejecting: haddock-library-1.1.1, 1.1.0 (conflict:
base==4.8.0.0/installed-901..., haddock-library => base>=4.3 && <4.8)
rejecting: haddock-library-1.0.1, 1.0.0 (conflict: pandoc =>
haddock-library>=1.1 && <1.2)
Backjump limit reached (change with --max-backjumps).
```

**Comment by lcycon** [comment]
You can append a "--reorder-goals" to cabal and in stalls from there. Fails later though with:

```
pandoc-1.6.0.1 failed during the configure step. The exception was:
user error ('/usr/local/bin/ghc' exited with an error:

/var/folders/6r/sm1nnrg553g4vzcmzvj5dvr40000gn/T/pandoc-1.6.0.1-11596/pandoc-1.6.0.1/dist/setup/setup.hs:95:18:
Not in scope: ‘catch’
)
```

**Comment by jgm** [comment]
+++ Luke Cycon [Apr 14 15 15:52 ]:
>You can append a "--reorder-goals" to cabal and in stalls from there. Fails later though with:
>
>```
>pandoc-1.6.0.1 failed during the configure step. The exception was:
>user error ('/usr/local/bin/ghc' exited with an error:

I'm sure you don't mean to be installing pandoc-1.6.  You can say
pandoc-1.13.2 to force the version.

**Comment by jgm** [comment]
Fixed by the new release of pandoc 1.13.2.1.
